apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -191,7 +197,7 @@ router.post(
       });
       return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
     } else if (req.body.__action === 'link_assessment') {
-      await queryAndLinkLineitem(instance, req.body.lineitem_id, req.body.assessment_id);
+      await queryAndLinkLineitem(instance, req.body.lineitem_id, req.body.unsafe_assessment_id);
```

Comment:
It wasn't obvious to me that `queryAndLinkLineitem` expected an unvalidated `assessment_id`; I had to trace things through to identify that it was ultimately validated. Two potential improvements here:

- In PT and increasingly in PL, we prefer to use an "options" object vs positional arguments so that we can see the name at the callsite.
- At the very least, the parameter in `queryAndLinkLineitem` should be renamed to `unsafe_assessment_id`.

Reply:
Renamed in 9b189f49a, we can revisit to switch to passing an object instead of positional arguments in a later PR, if needed.